HLTrigger/Muon/plugins/HLTTriMuonIsolation.h

```diff
@@ -92,9 +92,9 @@ HLTTriMuonIsolation::HLTTriMuonIsolation(const edm::ParameterSet& iConfig)
   produces<reco::CompositeCandidateCollection>("SelectedTaus");
 }
 
-HLTTriMuonIsolation::~HLTTriMuonIsolation() {}
```

Comment:
this should actually be defined `= default;` in the class body

Reply:
done